### ISSUE TITLE
first time running after announcement for v6.2.25

### ISSUE THREAD

**Issue by middleeast856** [goatcorp/FFXIVQuickLauncher#799]
System.AggregateException: One or more errors occurred. ---> System.ArgumentNullException: Value cannot be null.
Parameter name: source
   at System.Linq.Enumerable.Aggregate[TSource,TAccumulate](IEnumerable`1 source, TAccumulate seed, Func`3 func)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<CheckGateStatus>d__23.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 300
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<TryProcessLoginResult>d__25.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 581
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<Login>d__21.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 243
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<>c__DisplayClass20_0.<TryLogin>b__1() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 141
---> (Inner Exception #0) System.ArgumentNullException: Value cannot be null.
Parameter name: source
   at System.Linq.Enumerable.Aggregate[TSource,TAccumulate](IEnumerable`1 source, TAccumulate seed, Func`3 func)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<CheckGateStatus>d__23.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 300
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<TryProcessLoginResult>d__25.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 581
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<Login>d__21.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 243<---

Version: 6.2.25.0
Git Hash: 6.2.25-0-ge390c7f
Context: GetLoginFunc/Task
OS: Microsoft Windows NT 6.2.9200.0
64bit? True
DX11? True
Addons Enabled? True
Auto Login Enabled? True
Language: English
LauncherLanguage: English
Game path: E:\SquareEnix\FINAL FANTASY XIV - A Realm Reborn

**Comment by JDGBOLT** [comment]
I have the exact same error on mine as well trying to run xivlauncher in wine, though for me it's doing it every time I press the login button.

**Comment by reiichi001** [comment]
> > after i copied in the report here for the error and hit ok on the launcher it worked fine a second try had no errors at all.
> 
> sadly I'm still getting the error every time I try to log in.

If you uninstall/reinstall does it still happen?

**Comment by retep998** [comment]
Same error here:
```
System.AggregateException: One or more errors occurred. ---> System.ArgumentNullException: Value cannot be null.
Parameter name: source
   at System.Linq.Enumerable.Aggregate[TSource,TAccumulate](IEnumerable`1 source, TAccumulate seed, Func`3 func)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<CheckGateStatus>d__23.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 300
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<TryProcessLoginResult>d__25.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 581
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<Login>d__21.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 243
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<>c__DisplayClass20_0.<TryLogin>b__1() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 141
---> (Inner Exception #0) System.ArgumentNullException: Value cannot be null.
Parameter name: source
   at System.Linq.Enumerable.Aggregate[TSource,TAccumulate](IEnumerable`1 source, TAccumulate seed, Func`3 func)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<CheckGateStatus>d__23.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 300
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<TryProcessLoginResult>d__25.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 581
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<Login>d__21.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 243<---


Version: 6.2.25.0
Git Hash: 6.2.25-0-ge390c7f
Context: GetLoginFunc/Task
OS: Microsoft Windows NT 6.2.9200.0
64bit? True
DX11? True
Addons Enabled? True
Auto Login Enabled? True
Language: English
LauncherLanguage: English
Game path: E:\Steam\steamapps\common\FINAL FANTASY XIV Online
```

**Comment by retep998** [comment]
> > > after i copied in the report here for the error and hit ok on the launcher it worked fine a second try had no errors at all.
> > 
> > 
> > sadly I'm still getting the error every time I try to log in.
> 
> If you uninstall/reinstall does it still happen?

It still happens for me after uninstalling and reinstalling xivlauncher.

**Comment by Rocketboy227** [comment]
> 

I uninstalled XIVLauncher via the Control Panel and then reinstalled with the latest setup.exe, but I'm still experiencing the issue.

**Comment by Zeraphlol** [comment]
System.AggregateException: One or more errors occurred. ---> System.ArgumentNullException: Value cannot be null.
Parameter name: source
   at System.Linq.Enumerable.Aggregate[TSource,TAccumulate](IEnumerable`1 source, TAccumulate seed, Func`3 func)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<CheckGateStatus>d__23.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 300
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<TryProcessLoginResult>d__25.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 581
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<Login>d__21.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 243
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<>c__DisplayClass20_0.<TryLogin>b__1() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 141
---> (Inner Exception #0) System.ArgumentNullException: Value cannot be null.
Parameter name: source
   at System.Linq.Enumerable.Aggregate[TSource,TAccumulate](IEnumerable`1 source, TAccumulate seed, Func`3 func)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<CheckGateStatus>d__23.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 300
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<TryProcessLoginResult>d__25.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 581
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at XIVLauncher.Windows.ViewModel.MainWindowViewModel.<Login>d__21.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Windows\ViewModel\MainWindowViewModel.cs:line 243<---


Version: 6.2.25.0
Git Hash: 6.2.25-0-ge390c7f
Context: GetLoginFunc/Task
OS: Microsoft Windows NT 6.2.9200.0
64bit? True
DX11? True
Addons Enabled? True
Auto Login Enabled? True
Language: English
LauncherLanguage: English
Game path: C:\Program Files (x86)\SquareEnix\FINAL FANTASY XIV - A Realm Reborn

seems like the same error here as well. got this error after patching. tried the base xiv launcher and it seems to have patched properly. trying xivlauncher again just gives the same error.

**Comment by goaaats** [comment]
Fixed in 6.2.26, should be up in a few minutes. You can't play the game btw.
